cpp/include/cudf/io/csv.hpp

```diff
@@ -1332,7 +1332,7 @@ class csv_writer_options {
   size_type _rows_per_chunk = std::numeric_limits<size_type>::max();
   // character to use for separating lines (default "\n")
   std::string _line_terminator = "\n";
-  // character to use for separating lines (default "\n")
+  // character to use for separating column values (default ",")
```

Comment:
Does `column values` mean `rows` from the input column?

Reply:
There is no input column here.

Each line in the input CSV file corresponds to a row. The commas (or delimiters) separate the fields, i.e. values of each column.